### DISCUSSION TITLE
迅雷网盘开始限速

### DISCUSSION THREAD

**Discussion by nxjonson** [AlistGo/alist#6034] [Q&A]
大家的迅雷网盘，是不是alist网页内播放，还是webdav里播放都限速，只有一二百k？

**Comment by jonawang019** [comment]
我是晚上，慢得根本没法看，到11点会好一点

**Reply by nxjonson** [reply]
昨天莫名其妙又好了，看片2～3M

**Reply by pirate165** [reply]
是的，很玄幻有时候能看

**Comment by viqbgrg** [comment]
我是vip会员,速度有时候也可慢,一开始我以为是晚上高峰期的问题,最后我发现不同的资源,在同一时间的速度也是不一样的,我尝试改hosts把慢的域名改成快的那个,最后发现速度竟然也跟着快了起来,本来以为我看到了希望,用mosdns改hosts,结果发现有些可以用这个ip,有些不行
